pcdsdevices/valve.py

```diff
@@ -248,6 +248,14 @@ class VGC(VRC):
                                       doc='Downstream vacuum device used for'
                                       'interlocking this valve')
 
+    class VGC_2S(VGC):
```

Comment:
Error here prevents this file from working:
```suggestion
class VGC_2S(VGC):
```